src/app/dashboard/dashboard.component.ts

```diff
@@ -33,18 +30,11 @@ export class DashboardComponent implements OnInit {
         this.defaultAccount = defaultAccount;
         return this.auth.followItems('prx:series', {filters: 'v4', zoom: 'prx:image'});
```

Comment:
I think if this zoom was `prx:image,prx:distributions` that would cut down on 10 more requests.